build.ps1

```diff
@@ -241,16 +241,14 @@ $CAKE_EXE_INVOCATION = if ($IsLinux -or $IsMacOS) {
 }
 
 # Build Cake arguments
-$cakeArguments = ""
-If ($Script) {
-    $cakeArguments += @("`"$Script`"");
-}
+$cakeArguments = @()
+if ($Script) { $cakeArguments += "`"$Script`"" }
 if ($Target) { $cakeArguments += "-target=`"$Target`"" }
 if ($Configuration) { $cakeArguments += "-configuration=$Configuration" }
 if ($Verbosity) { $cakeArguments += "-verbosity=$Verbosity" }
 if ($ShowDescription) { $cakeArguments += "-showdescription" }
 if ($DryRun) { $cakeArguments += "-dryrun" }
-$cakeArguments += $ScriptArgs
+if ($ScriptArgs) { $cakeArguments += "$ScriptArgs" }
```

Comment:
Why is this change being made? `$cakeArguments` and `$ScriptArgs` are both arrays. Isn't concatenating the arrays (as the original line does) the right thing to do?

Reply:
@jnm2 You are right - just changed it back to original.